Formula/clucene.rb

```diff
@@ -39,8 +39,19 @@ class Clucene < Formula
   end
 
   def install
+    args = %W[
+      -DLIB_DESTINATION=#{lib}
+      -DBUILD_CONTRIBS_LIB=ON
```

Comment:
Why do we need to set these?

Reply:
The "contribs-lib" was a part of CLucene but not automatically enabled while configuring, `libclucene-contribs-lib.dylib` can be compiled by adding `BUILD_CONTRIBS`.
Specifying `LIB_DESTINATION` to avoid wrong library install path

Reply:
If it's not automatically enabled upstream generally doesn't think it's ready for the general public.

As for the wrong install path, is that actually a problem or preventative?